app/src/main/res/values/theme.xml

```diff
@@ -80,6 +81,7 @@
     <attr name="textColorThird" format="reference" />
     <attr name="colorPlaceholder" format="reference" />
     <attr name="colorWarning" format="reference" />
+    <attr name="colorTooltip" format="reference" />
```

Comment:
I think we can access `@color/colorTooltip` directly instead, we're not gaining any particular benefits from using this indirection